neat_cache/lib/src/providers/resp.dart

```diff
@@ -203,7 +203,7 @@ class RespClient {
   Future<void> close({bool force = false}) async {
     _closing = true;
 
-    if (!_closing) {
+    if (!force) {
```

Comment:
Hmm, I think the dartdoc could b e improved. Perhaps something like:

```
///  Close the connection to redis and send a `QUIT` command unless `force`.
```

Otherwise we should perhaps just always send it?